src/config/markets.ts

```diff
@@ -147,3 +147,26 @@ export function getMarketListingDate(chainId: number, marketAddress: string): Da
 
   return tokenListing.listingDate;
 }
+
+export const GLV_MARKETS_ENABLED = {
```

Comment:
maybe just GLV_ENABLED? because each glv market can potentially be disabled, might be confusing